src/settings/settingsdialog.cpp

```diff
@@ -105,6 +109,9 @@ SettingsDialog::SettingsDialog(MainWindow *parent)
 
     ui->ocrLanguagesListWidget->addLanguages(parent->ocr()->availableLanguages());
 
+    for (const QOnlineTranslator::Language configurableLang : QOnlineTts::validRegions().keys())
```

Comment:
We usually don't put const qualifiers on trivial types in loop. Let's keep it consistent, I will revisit it later: 
```suggestion
    for (QOnlineTranslator::Language configurableLang : QOnlineTts::validRegions().keys())
```